examples/mlflow/keras_mlflow.py

```diff
@@ -83,18 +89,12 @@ def objective(trial):
               epochs=EPOCHS,
               verbose=False)
 
-    score = model.evaluate(X_test, y_test, verbose=0)
-
-    with mlflow.start_run():
-        mlflow.log_params(trial.params)
-        mlflow.log_metrics({'mean_squared_error': score})
-
-    return score
+    return model.evaluate(X_test, y_test, verbose=0)
 
 
 if __name__ == '__main__':
-    study = optuna.create_study(direction='minimize')
-    study.optimize(objective, n_trials=100, timeout=600)
+    study = optuna.create_study(study_name='keras-wine', direction='minimize')
```

Comment:
I think 'study_name' argument can be removed since this example doesn't use an RDB storage.

Reply:
Thank you for your comment.
I set it to show somewhat meaningful names rather than `noname-xxx` in MLflow UI.But we can distinguish studies if we remove the name. So I'll rwmove it.

Reply:
Fixed: https://github.com/optuna/optuna/pull/799/commits/ef61f4150584d5f6ffd11e05c8090a86e1857f0e

I also removed `direction` option from `create_study` because the default direction is `minimize` in the commit https://github.com/optuna/optuna/pull/799/commits/5c4942ccbf5bc7827f6a6bf979aa4626617c5780.

The new screenshot of mlflow ui is as follows:

![image](https://user-images.githubusercontent.com/3255979/71046991-71accc80-217d-11ea-9109-eb46d4ee916e.png)

Reply:
Thanks.

This is just a thought. Current default study names are not human-friendly, so it might worth to improve the name generation logic in the future (e.g., adopt the naming rule being used by Docker).

Reply:
I agree with you. I posted an issue about the name generation as  https://github.com/optuna/optuna/issues/803. Thank you.

By the way, the following part of my comment https://github.com/optuna/optuna/pull/799#discussion_r358699168 is not true because all studies created with `InMemoryStorage` have the same default name (i.e., `no-name-00000000-0000-0000-0000-000000000000`). I'm sorry to give you a wrong information.

> But we can distinguish studies if we remove the name.

Reply:
> I posted an issue about the name generation as #803.

Thanks!

> all studies created with InMemoryStorage have the same default name (i.e., no-name-00000000-0000-0000-0000-000000000000).

Oops. That is, there is no actual merit to use the default name...?

Reply:
That's right. Maybe storage classes should share the name generator instead of adding name generation code for `InMemoryStorage` to this example.